cmd/tapcli/assets.go

```diff
@@ -304,12 +323,10 @@ func mintAsset(ctx *cli.Context) error {
 
 	resp, err := client.MintAsset(ctxc, &mintrpc.MintAssetRequest{
 		Asset: &mintrpc.MintAsset{
-			AssetType: assetType,
-			Name:      ctx.String(assetTagName),
-			AssetMeta: assetMeta,
-			DecimalDisplay: uint32(
-				ctx.Uint64(assetDecimalDisplayName),
-			),
+			AssetType:       assetType,
+			Name:            ctx.String(assetTagName),
+			AssetMeta:       assetMeta,
+			DecimalDisplay:  uint32(decDisplay),
```

Comment:
Based on this comment, I think we need another check above to assume that if decimal display is set, then we're using a JSON meta type.

Reply:
Or we just update in the docs that decimal display requires the json meta data type?